gatsby-browser.js

```diff
@@ -1,5 +1,23 @@
 /* eslint-env node */
 
 const PageWrapper = require('./src/components/PageWrapper').default
+const { getDetails, setDetails } = require('./src/utils/shared/localStorage')
 
 exports.wrapPageElement = PageWrapper
+
+exports.onPreRouteUpdate = ({ location, prevLocation }) => {
```

Comment:
can we make collapsible itself saving this on the expnasion? don't like that we do this global search on every navigate